src/eventEmitter/index.js

```diff
@@ -0,0 +1,165 @@
+var
+  uuid = require('uuid');
+
+function EventEmitter(eventTimeout) {
+  Object.defineProperties(this, {
+    eventListeners: {
+      value: {}
+    },
+    eventTimeout: {
+      value: eventTimeout || 200,
+      writeable: false
+    }
+  });
+}
+
+/**
+ * Emit an event to all registered listeners
+ * An event cannot be emitted multiple times before a timeout has been reached.
+ */
+EventEmitter.prototype.emitEvent = function emitEvent(event) {
+  var
+    self = this,
+    now = Date.now(),
+    args = Array.prototype.slice.call(arguments, 1),
+    eventProperties = this.eventListeners[event];
+
+  if (!eventProperties) {
+    return false;
+  }
+
+  if (eventProperties.lastEmitted && eventProperties.lastEmitted >= now - this.eventTimeout) {
```

Comment:
The code here is quite light, and anyway, we would have to override all library's methods to manage the `eventTimeout` behaviour.
So the question is: do we still need this behaviour or not ? 
If yes, I rather reimplement the whole methods like here
If not, of course, we can simply remove it and use `ev-emitter` instead.